exchanges/binance/binance_websocket.go

```diff
@@ -258,6 +258,8 @@ func (b *Binance) wsHandleData(respRaw []byte) error {
 					Date:            data.Data.OrderCreationTime,
 					Pair:            p,
 					ClientOrderID:   clientOrderID,
+					Cost:            data.Data.Commission,
+					CostAsset:       currency.NewCode(data.Data.CommissionAsset),
```

Comment:
This change has broken `TestWsOrderExecutionReport`. Please check whether `data.Data.CommissionAsset != ""` first before creating the currency code.
Could you please also update the test? The json has `"n":"0", "N": null,`. If you could update it to have values and compare them that would be much appreciated.